qcelemental/molparse/to_schema.py

```diff
@@ -36,11 +36,14 @@ def to_schema(molrec: Dict, dtype: str, units: str='Bohr', *, np_out: int=False)
     """
     qcschema: Dict = {}
 
-    if molrec['units'] == 'Angstrom' and units == 'Bohr' and 'input_units_to_au' in molrec:
-        factor = molrec['input_units_to_au']
+    if units is None:
+        geom = np.array(molrec["geom"], copy=copy)
```

Comment:
`factor` not getting defined in this block, which seems bad.

Reply:
It is not being used anywhere else?

Reply:
I think there'll be trouble in the case that you pass in `units=None, dtype=2` but molrec for this case happens to be Angstrom. As currently coded, you'll get an Ang `models.Molecule` 😱

Reply:
Ah, ok. Let me try another way. Are we being too aggressive/dangerous in other optimization areas?

Reply:
you mean in adding a `copy=False` option?

Reply:
Other areas where we prevent copying as well.